docs/editions.rst

```diff
@@ -94,31 +115,59 @@ CrateDB Community Edition
 The CrateDB *Community Edition* (CrateDB CE) does not include any
 :ref:`enterprise-features` but can be run on as many nodes as you wish.
 
-CrateDB CE must be built from source, like so:
+.. NOTE::
+
+   CrateDB requires a `Java virtual machine`_ to run.
+
+   Starting with CrateDB 4.2, a JVM is bundled with the tarball and no
+   extra installation is necessary.
+
+   Versions starting from 3.0 to 4.1 require a `Java 11`_ installation. We
+   recommend using `Oracle's Java`_ on macOS and OpenJDK_ on Linux Systems.
+
+   Earlier versions required Java 8.
+
+CrateDB CE must be built from source:
 
 .. code-block:: console
 
    sh$ git clone https://github.com/crate/crate
    sh$ cd crate
-   sh$ git submodule update --init
```

Comment:
Maybe re-add this as optional step after the `git checkout <tag>` command if a version earlier than 4.0 is checked out, in which case this would still be necessary

Reply:
Hmmmm, in that case, I am thinking that it might be useful to include it as a note.  

This is based on me noticing that we talk about the installation requirements of older versions here: https://crate.io/docs/crate/tutorials/en/latest/install-run/basic.html

Are you still indifferent about it?

Reply:
Ah good point.  The docs in this repo are versioned, the docs in the tutorials or how-to repo aren't.  

So +1 for removing the submodule hint completely